packages/account-fs/agent/agent.js

```diff
@@ -229,6 +233,34 @@ export const StorageCapability = {
     return newContent
   },
 
+  async syncCarFileWithHub() {
+
+    const generateCarFile = async (cid) => {
```

Comment:
we can also pass the CID we get from hub, so that we can only generate CAR buffer till that CID - as we can assume rest of the CIDs are already present on hub